src/components/Input/InputField/WebsiteInputField.tsx

```diff
@@ -30,8 +30,17 @@ const WebsiteInputField = ({
           name="website-url"
           value={inputUrl}
           placeholder="Enter a web page URL"
+          pattern="^(https?|ftp):\/\/[^\s/:?#][^\s]*$" // Regex to check if the input is a valid URL
           required
-          onChange={(e) => setInputUrl(e.target.value)}
+          onChange={(e) => {
+            // This will display the error message on browser's default tooltip
+            if (!e.target.validity.valid) {
+              e.target.setCustomValidity("Please enter a valid URL");
```

Comment:
Oh didn't know we could do that. We should probably use the browser's validity tooltips to be more consistent :). I can edit my PR tomorrow to use the default tooltips